pogoiv/cli.py

```diff
@@ -65,13 +65,38 @@ def main():
     parser.add_argument('--powered', nargs="+", choices=["True", "False"], required=True, help='For this data point, has the Pokemon ever been powered?')
     parser.add_argument('-p', '--pokemon', required=True, help='Name of the Pokemon')
 
+    parser.add_argument(
+        '--a1', '--appraisal1',
```

Comment:
I'd prefer to see something like --ao and --appraisal-overall for the top level item and --ap / --appraisal-stats for the stat specific appraisal. Code is more readable when talking about things of similar but slightly different descriptions when using words over numbers.